content/messaging/references/default-properties/index.mdx

```diff
@@ -0,0 +1,495 @@
+
+|               |                                         |
+| ------------- | --------------------------------------- |
+| **Access**    | `rw` for Customers in chats and threads |
+| **Data type** | `string`                                |
+
+</Text>
+<Code>
+<CodeResponse title={'Properties in the lc2 namespace'}>
+
+```json
+{
+  "properties": {
+      "lc2": {
+          "file_safety_confirmation": true,
+          "form_type": "prechat",
+          "greeting_id": 2,
+          "greeting_unique_id": "Q10X0W041P",
+          "prechat_group_id": 0,
+          "welcome_author_id": "bbb67d600796e9f277e360e842418833",
+          "welcome_message": true,
+          "welcome_text": "Welcome! How can I help you today?"
+      }
+  }
+}
+```
+
+</CodeResponse>
+
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+## rating
+
+The properties describing chat rating.
+
+### comment
+
+Contains the text message rating left by the Customer.
+
+|               |                                                                     |
+| ------------- | ------------------------------------------------------------------- |
+| **Access**    | `r` for Agents and `rw` for Customers in chats, threads, and events |
+| **Data type** | `string`                                                            |
+
+### score
+
+Describes chat rating: `0` - thumbs down, `1` - thumbs up.
+
+|               |                                                                                                                         |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| **Access**    | <ul><li>`r` for Agents in chats, threads, and events</li><li>`rw` for Customers in chats, threads, and events</li></ul> |
+| **Data type** | `int`                                                                                                                   |
+| **Values**    | `0` `1`                                                                                                                 |
+
+</Text>
+<Code>
+<CodeResponse title={'Properties in the rating namespace'}>
+
+```json
+{
+  "properties": {
+      "rating": {
+          "comment": "Action completed",
+          "score": 0
+      }
+  }
+}
+```
+
+</CodeResponse>
+
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+## routing
+
+### continuous
+
+Specifies if the continuous messaging mode is enabled.
+
+|               |                                                   |
+| ------------- | ------------------------------------------------- |
+| **Access**    | `r` for Agents and Customers in chats and threads |
+| **Data type** | `bool`                                            |
+
+### email_follow_up
+
+Specifies if there should be an email follow-up of the message left by the Customer. For example, when a Customer left a message because they were no online Agents.
```

Comment:
I think it'd be nice to add that the email will be sent if messaging mode is enabled on the license